src/controllers/notificationController.ts

```diff
@@ -31,7 +37,29 @@ export default class NotificationController {
   > =>
     withValidatedOrValidationError(
       Notification.decode(req.body),
-      this.notificationService.notify
+      async (data: Notification) => {
+        const userHasSessionOrError = await this.sessionStorage.userHasActiveSessions(
+          data.message.fiscal_code
+        );
+        if (isRight(userHasSessionOrError) && userHasSessionOrError.value) {
+          return this.notificationService.notify(data);
+        }
+        const newData = {
+          ...data,
+          message: {
+            ...data.message,
+            content: {
+              ...data.message.content,
+              subject: "Entra nell'app per leggere il contenuto" as string &
```

Comment:
maybe it's worth to use an enviroment / configuration variable for subject here (what about translations ?)

Reply:
cc @matteodesanti